src/lib/journey/callbacks/password/validated-create-password.svelte

```diff
@@ -1,32 +1,48 @@
 <script lang="ts">
   import type { ValidatedCreatePasswordCallback } from '@forgerock/javascript-sdk';
 
+  import { getValidationFailures } from '$journey/callbacks/_utilities/callback.utilities';
   import Base from '$journey/callbacks/password/base.svelte';
-  import {
-    getPasswordValidationFailureText,
-    isInputRequired,
-  } from '$journey/utilities/callback.utilities';
+  import { isInputRequired } from '$journey/callbacks/_utilities/callback.utilities';
+  import Policies from '$journey/callbacks/_utilities/policies.svelte';
 
   export let callback: ValidatedCreatePasswordCallback;
   export let firstInvalidInput: boolean;
   export let idx: number;
 
   /**
-   * This callback does not indicate that the input is required.
+   * At the time of this writing, this callback is never marked as required,
+   * but I'm adding this here as that could change.
    */
   const isRequired = isInputRequired(callback);
 
-  let label = callback.getPrompt();
-  let validationFailure = getPasswordValidationFailureText(callback, label);
+  let inputName = callback?.payload?.input?.[0].name || `password-${idx}`;
+  let prompt = callback.getPrompt();
+
+  let validationFailures = getValidationFailures(callback, prompt);
+  let isInvalid = !!validationFailures.length;
 
   $: {
     /**
      * We need to wrap this in a reactive block, so it reruns the function
      * on value changes within `callback`
      */
-    label = callback.getPrompt();
-    validationFailure = getPasswordValidationFailureText(callback, label);
+    inputName = callback?.payload?.input?.[0].name || `password-${idx}`;
+    prompt = callback.getPrompt();
+
+    validationFailures = getValidationFailures(callback, prompt);
+    isInvalid = !!validationFailures.length;
   }
 </script>
 
-<Base {callback} {firstInvalidInput} {idx} {isRequired} {validationFailure} />
+<Base
+  {callback}
+  {firstInvalidInput}
+  {idx}
+  {isInvalid}
+  {isRequired}
+  key={inputName}
+  showMessage={!validationFailures.length}
+>
+  <Policies {callback} key={inputName} label={prompt} messageKey="passwordRequirements" />
```

Comment:
Here's the usage of the new Policies component. Notice how we deactivate the simple message (line above) that comes with the Base component if there are validation failures and use the more complex Policies component.